modules/ismp/clients/parachain/inherent/src/lib.rs

```diff
@@ -68,6 +68,7 @@ impl ConsensusInherentProvider {
         let message = ConsensusMessage {
             consensus_state_id: ismp_parachain::consensus::PARACHAIN_CONSENSUS_ID,
             consensus_proof: consensus_proof.encode(),
+            signer: Default::default(),
```

Comment:
I would assume there is no signer since this is coming from the relay chain.

Reply:
Yes this is correct

Reply:
That could make sense. However, I am not sure from where I could get the block author's public key. From the digest?

Reply:
Yeah I'd like to believe that there's a runtime or keystore API for this.